stryker.jest.conf.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I think this whole file is not really needed.

Reply:
I was trying to run stryker using the "native" `jestRunner` for improved performance. But I was not able to make it work, so I agree it should not be part of this PR.